src/components/trade/LimitOrder/LimitOrder.stories.tsx

```diff
@@ -1,41 +1,31 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Meta, Story } from '@storybook/react/types-6-0'
 import { LimitOrder, Props } from './LimitOrder'
 import BigNumber from 'bignumber.js'
+import { GNO, DAI } from 'storybook/tokens'
 
 export default {
   title: 'Trade/LimitOrder',
 } as Meta
 
-const GNO = {
-  id: 1,
-  address: '0x1',
-  symbol: 'GNO',
-  decimals: 18,
-}
-
-const DAI = {
-  id: 2,
-  address: '0x2',
-  symbol: 'DAI',
-  decimals: 18,
-}
-
 const defaultProps = {
-  sellToken: GNO,
-  receiveToken: DAI,
+  sellToken: DAI,
+  receiveToken: GNO,
   limitPrice: new BigNumber('55.247234'),
   amount: '100',
-  isPriceInverted: false,
 }
 
 const Template: Story<Partial<Props>> = (props) => {
+  const [isPriceInverted, setIsPriceInverted] = useState<boolean>(false)
+
   return (
     <LimitOrder
+      isPriceInverted={isPriceInverted}
       onSwapPrices={(): void => {
         console.log('[LimitOrder.story] Swap Prices')
+        setIsPriceInverted(!isPriceInverted)
```

Comment:
`setIsPriceInverted(state => !state)`

Reply:
same thing, isn't it?
I'll use yours, since it's more "pure"